x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -241,34 +257,43 @@ export const signalRulesAlertType = ({
           }
 
           logger.debug(
-            `Finished signal rule name: "${name}", id: "${alertId}", rule_id: "${ruleId}", output_index: "${outputIndex}"`
+            `Finished signal rule name: "${name}", id: "${alertId}", rule_id: "${ruleId}"`
           );
           await writeCurrentStatusSucceeded({
             services,
             currentStatusSavedObject,
+            bulkCreateTimes: creationSucceeded.bulkCreateTimes,
+            searchAfterTimes: [...creationSucceeded.searchAfterTimes],
+            lastLookBackDate: creationSucceeded.lastLookBackDate?.toISOString(),
           });
         } else {
           await writeSignalRuleExceptionToSavedObject({
             name,
             alertId,
             currentStatusSavedObject,
             logger,
-            message: `Bulk Indexing signals failed. Check logs for further details Rule name: "${name}" id: "${alertId}" rule_id: "${ruleId}" output_index: "${outputIndex}"`,
+            message: `Bulk Indexing signals failed. Check logs for further details \nRule name: "${name}"\nid: "${alertId}"\nrule_id: "${ruleId}"\n`,
             services,
             ruleStatusSavedObjects,
             ruleId: ruleId ?? '(unknown rule id)',
+            bulkCreateTimes: creationSucceeded.bulkCreateTimes,
+            searchAfterTimes: [...creationSucceeded.searchAfterTimes],
```

Comment:
no need of the spreading here
```suggestion
            searchAfterTimes: creationSucceeded.searchAfterTimes,
```